<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kournable</class>
 <widget class="QWidget" name="Kournable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1334</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kournable</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="weekdayLayout">
     <item>
      <layout class="QVBoxLayout" name="mondayLayout">
       <item>
        <widget class="QLabel" name="mondayLabel">
         <property name="text">
          <string>Monday</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="mondayPlainTextEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="tuesdayLayout">
       <item>
        <widget class="QLabel" name="tuesdayLabel">
         <property name="text">
          <string>Tuesday</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="tuesdayPlainTextEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="wednesdayLayout">
       <item>
        <widget class="QLabel" name="wednesdayLabel">
         <property name="text">
          <string>Wednesday</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="wednesdayPlainTextEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="thursdayLayout">
       <item>
        <widget class="QLabel" name="thursdayLabel">
         <property name="text">
          <string>Thursday</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="thursdayPlainTextEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="fridayLayout">
       <item>
        <widget class="QLabel" name="fridayLabel">
         <property name="text">
          <string>Friday</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="fridayPlainTextEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="weekendLayout">
       <item>
        <widget class="QLabel" name="weekendLabel">
         <property name="text">
          <string>Weekend</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="weekendPlainTextEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="monthLayout">
     <item>
      <layout class="QVBoxLayout" name="thisMonthLayout">
       <item>
        <widget class="QLabel" name="thisMonthLabel">
         <property name="text">
          <string>This Month</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="thisMonthPlainTextEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="nextMonthLayout">
       <item>
        <widget class="QLabel" name="nextMonthLabel">
         <property name="text">
          <string>Next Month</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="nextMonthPlainTextEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
